lib/miq_automation_engine/service_models/miq_ae_service_container_project.rb

```diff
@@ -0,0 +1,18 @@
+module MiqAeMethodService
+  class MiqAeServiceContainerProject < MiqAeServiceModelBase
+    expose :container_groups,       :association => true
+    expose :container_routes,       :association => true
+    expose :container_replicators,  :association => true
+    expose :container_services,     :association => true
+    expose :container_definitions,  :association => true
+    expose :container_nodes,        :association => true
+    expose :container_quotas,       :association => true
+    expose :container_quota_items,  :association => true
+    expose :container_limits,       :association => true
+    expose :container_limit_items,  :association => true
+    expose :container_builds,       :association => true
+    expose :metrics,                :association => true
+    expose :metric_rollups,         :association => true
+    expose :vim_performance_states, :association => true

```

Comment:
@supernoodz Is there really a need to expose all of the relationships on the model?  If so you will also need to create corresponding service_models for each that does not already exist.

We do not expose `metrics` on other models, is there a use-case you are thinking of?

Also, is there a BZ or other tracker for this enhancement.